azure-pipelines.yml

```diff
@@ -467,6 +467,35 @@ jobs:
     parameters:
       includePatterns:
         - "/hw/***"
+  - task: DownloadSecureFile@1
+    name: gcpkey
+    inputs:
+      secureFile: gcpkey.json
+  - bash: |
+      echo "Installing gsutil"
+      sudo apt-get install -y apt-transport-https ca-certificates gnupg
+      echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" \
+        | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
```

Comment:
Would something like this improve debuggability?
```sh
sudo cat >> /etc/apt/... <<< "deb ... "
cat /etc/apt/...
```
This way the whole file gets printed, to show what else was there in addition to what we appended.

Reply:
Added a cat to print out the entire line.
After some experimentation, I decided to stick with the `echo "..." | sudo tee ...` approach. The redirection was not working well with sudo and I think the sudo tee approach is more clear on what it's trying to do.